README.md

```diff
@@ -37,14 +36,20 @@ bazel build server \
 
 ## Java support
 
-If you need Java support, you just need to add a few more flags:
+Bazel provides support for Java toolchains out of the box.
+You can enabled the Java toolchain with the following flags:
 
 ```
---javabase=@buildbuddy_toolchain//:javabase
---host_javabase=@buildbuddy_toolchain//:javabase
---java_toolchain=@buildbuddy_toolchain//:java_toolchain
+--java_language_version=11
```

Comment:
11 is super old at this point, should we recommend 17 in the defaults?

Reply:
s/11/17/ sgtm. Applied